anecdote/metadata.csv

```diff
@@ -0,0 +1,31 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Unfortunately, we do not support `-` characters in the metrics. Would you be able to rename the metric names that have `-` and change it to an `_` or `.`?

Reply:
I pushed the changes

Reply:
There still appear to be metrics with a `-`:
`anecdote.feedback.custom-csv.bug`
and
`anecdote.feedback.custom-parquet.bug`

Reply:
Could you have a look again, please?